src/object/srv_obj.c

```diff
@@ -1338,7 +1338,6 @@ obj_ioc_fini(struct obj_io_context *ioc)
 		ds_cont_child_put(ioc->ioc_coc);
 		ioc->ioc_coc = NULL;
 	}
-	ioc->ioc_map_ver = 0;
```

Comment:
Could you explain why it's matter? It looks to me the obj_io_context is finalized after use here.

Reply:
in obj_ioc_begin(), if the pool map version does not match, it will return ESTALE, then obj_ioc_fini() is called, where ioc_map_ver is reset to 0, but later in obj_rw_reply, the map_ver needs to be replied to client. So client needs this information to know whether the pool map needs to be refreshed.

Reply:
You mean obj_rw_reply() replied 0 map version set by obj_ioc_fini()? I didn't see how.

Reply:
I see, an obj_ioc_fini() is called inside of obj_ioc_begin().